pkg/service/sniffer/runtime/docker.go

```diff
@@ -29,7 +29,7 @@ func (d DockerBridge) BuildTcpdumpCommand(containerId *string, netInterface stri
 	d.tcpdumpContainerName = "ksniff-container-" + utils.GenerateRandomString(8)
 	containerNameFlag := fmt.Sprintf("--name=%s", d.tcpdumpContainerName)
 
-	command := []string{"docker", "--host", "unix:///host/var/run/docker.sock",
+	command := []string{"docker", "--host", "unix:///host/run/docker.sock",
```

Comment:
Would you mind removing this change for now? I think we're still trying to decide the best solution in #82

Reply:
Of course, no problem.